prac2-pages/anshumali_suri/lib/main.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  bool value=false;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+        padding: const EdgeInsets.all(20),
+        child: ListView(
+          children: <Widget>[
+
+            Container(
```

Comment:
Try to minimize the use of Container in your Flutter projects, they only serve to increasing complexity if you are not going to specify any other parameters. 
Moreover, since you have not specified the dimensions of the Container (i.e. its height and width), alignment won't work.

Try to change your code such that you make minimal use of Containers while also not losing any of the functionality and features.

Reply:
Resolved in commit  [Reduced usage of Container widget](https://github.com/RitwikSingh28/GitHubPrac/pull/15/commits/480ac135a3b666197b5d3f6400c14416f023cb06)